### DISCUSSION TITLE
Jupyter notebooks are not displaying any cells on Chrome on macOS

### DISCUSSION THREAD

**Discussion by kotaishr** [community/community#17588] [Codespaces]
Jupyter notebooks are not displaying any cells on Chrome on macOS.  I can avoid this problem by using Safari.

<img width="615" alt="image" src="https://user-images.githubusercontent.com/16282891/171327290-8151392e-8027-437f-81c7-ba1eb0d15f9f.png">

Versions
- Chrome: Version 102.0.5005.61 (Official Build) (x86_64)
- macOS: version 12.2.1 (21D62)

Is anyone having similar issue?

**Comment by thomasXwang** [comment]
Hi, I'm having the same issue
I first managed to render the Notebook by chance after a few tries, and used it successfully.
After this, it has kept rendering nothing at all, just like on your screenshot :(
It has occurred to me both on Firefox and Chrome on Debian

**Comment by osortega** [comment]
Hello! 👋 
Thank you for reporting! I've been looking into this but I'm unable to reproduce the problem on my end.
Is it possible for you to share the repository where you are seeing this issue?

I'm also curious on what happens if you click on the `+ Code` button, does that create a cell at all?

**Reply by kotaishr** [reply]
Thank you for the reply!

This is the repository (I created a new minimal repo) : https://github.com/kotaishr/test_codespaces

Clicking `+Code` button doesn't change the view (right pane in the image below), but it seems a new cell is added in the file (left pane, opening the same file a.ipynb with text editor).

<img width="956" alt="image" src="https://user-images.githubusercontent.com/16282891/173262975-acc63dc3-47b2-41ba-bb60-75db0d535029.png">
